Index: muffin-2.91.92/configure.in
===================================================================
--- muffin-2.91.92.orig/configure.in
+++ muffin-2.91.92/configure.in
@@ -485,11 +485,6 @@
     esac
 
     case " $CFLAGS " in
-    *[\ \	]-Wcast-align[\ \	]*) ;;
-    *) CFLAGS="$CFLAGS -Wcast-align" ;;
-    esac
-
-    case " $CFLAGS " in
     *[\ \	]-Wsign-compare[\ \	]*) ;;
     *) CFLAGS="$CFLAGS -Wsign-compare" ;;
     esac
